server/routes/address.js

```diff
@@ -80,4 +106,38 @@ function isEmptySubmission(userInput) {
     return !addressLine1 && !addressTown && !postCode;
 }
 
+function getCurfewAddressChoice(licence) {
+    if (isYes(licence, ['proposedAddress', 'optOut', 'decision'])) {
+        return 'OptOut';
+    }
+
+    if (isYes(licence, ['proposedAddress', 'addressProposed', 'decision'])) {
+        return 'Address';
+    }
+
+    if (isYes(licence, ['bassReferral', 'bassRequest', 'bassRequested'])) {
```

Comment:
Like the function. We should probably use elsewhere as and when we touch appropriate files